doc/redeemer.md

```diff
@@ -0,0 +1,81 @@
+# Ticket Redemption Service
+
+The ticket redemption service allows an Orchestrator to run a separate service for redeeming winning tickets on-chain using a separate ETH account. 
+
+The ticket redemption service is responsible for redeeming winning tickets as well as pushing max float updates about broadcasters back to the orchestrator(s). 
+
+## TicketQueue
+
+1. The `ticketQueue` is a loop that runs everytime a new block is seen. It will then pop tickets off the queue starting with the oldest ticket first, and sends it to the `LocalSenderMonitor` for redemption if the `recipientRand` for the ticket has expired. 
+
+2. When the `LocalSenderMonitor` receives a ticket from the `ticketQueue` it will substract `ticket.faceValue` from the outstanding `maxFloat` as long as the ticket is in limbo. 
```

Comment:
pending between the Redeemer and the ethereum node 

This can be either pending on-chain or an HTTP request to the eth node that has some latency to it.